src/config.rs

```diff
@@ -152,6 +153,12 @@ fn default_daemon_dir() -> PathBuf {
     home
 }
 
+fn default_blocks_dir(daemon_dir: &PathBuf) -> PathBuf {
+    let mut blocks_dir = daemon_dir.clone();
+    blocks_dir.push("blocks");
```

Comment:
This is slightly more complicated than it could be. I suggest accepting `&Path` instead of `&PathBuf` and then just using `join()` In other words, this:

```rust
fn default_blocks_dir(daemon_dir: &Path) -> PathBuf {
    daemon_dir.join("blocks")
}
```

Reply:
Right, I just moved the code from `list_blk_files()`. But other words is way nicer, thanks !